CONTRIBUTING.md

```diff
@@ -185,14 +189,49 @@ def do_something(value: bool = False) -> bool:
 
 Test code and branch coverage is already tracked using [coverage](https://github.com/nedbat/coveragepy) and the [pytest-cov](https://github.com/pytest-dev/pytest-cov) plugin for pytest, and it measures how much code in the `src/macaron/` folder is covered by tests.
 
-You can also run integration tests locally:
+#### Integration tests
+
+To know more about our integration test utility, please see [here](./tests/integration/README.md).
+
+We have two types of integration tests:
+1. Integration tests that are run against the Macaron Python package.
+  - Prerequisites: The dev environment must be setup first (you should only need to run once, unless you make changes to the project's dependencies).
+```bash
+make setup
+```
+  - Run these tests locally with:
 ```bash
 make integration-test
 ```
+2. Integration tests that are run against the container image.
+  - Prerequisites: Setup the integration test utility for docker.
+```bash
+make setup-integration-test-utility-for-docker
```

Comment:
This is not needed. `make setup` will install all the required dependencies.

Reply:
I have removed it in https://github.com/oracle/macaron/pull/815/commits/bbc1a27658cc537a70e7f76838be55adbb9fd19b